exp/addons/api/v1beta1/v1beta2_condition_consts.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+
+// Conditions that will be used for the ClusterResourceSet object in v1Beta2 API version.
+const (
+	// ClusterResourceSetReadyV1Beta2Condition is true if the ClusterClass is not deleted,
```

Comment:
```suggestion
	// ClusterResourceSetReadyV1Beta2Condition is true if the ClusterResourceSet is not deleted,
```
But not sure if we want to add Ready now. I personally wouldn't do it.

Reply:
Dropped